features/discover/src/main/java/com/chesire/nekome/app/discover/DiscoverViewModel.kt

```diff
@@ -55,4 +63,23 @@ class DiscoverViewModel @ViewModelInject constructor(
      */
     val trendingManga: LiveData<AsyncState<List<SeriesModel>, DiscoverError>>
         get() = _trendingManga
+
+    private fun TrendingEntity.toSeriesModel(): SeriesModel = SeriesModel(
```

Comment:
Create an issue to hand this over to a repository or something, or at least a converter class